pepper-apis/src/main/java/org/broadinstitute/ddp/json/Profile.java

```diff
@@ -33,10 +36,13 @@
     private String firstName;
     @SerializedName(LAST_NAME)
     private String lastName;
+    @SerializedName(BIRTH_DATE)
+    private String birthDate;
 
     public Profile(UserProfile other) {
         LocalDate birthDate = other.getBirthDate();
         UserProfile.SexType sexType = other.getSexType();
+        this.birthDate = birthDate != null ? birthDate.toString() : null;
```

Comment:
Is there a reason why we're converting the date to a string instead of storing a reference to the source object and performing the conversion as needed?

Reply:
I am doing that to Serialize and deserialize. 
@SerializedName(BIRTH_DATE)
private String birthDate;